services/core-web/src/components/Forms/ExplosivesPermit/MagazineForm.js

```diff
@@ -134,7 +134,7 @@ export class MagazineForm extends Component {
                 label="Longitude *"
                 id={`${field}longitude`}
                 name={`${field}longitude`}
-                validate={[number, maxLength(12), lon, required]}
+                validate={[number, maxLength(12), lonNegative, required]}
```

Comment:
Think this needs to have both `lonNegative` and `lon` to make sure it is both a valid longitude and is negative